worlds/yoshisisland/docs/setup_en.md

```diff
@@ -0,0 +1,122 @@
+# Yoshi's Island Archipelago Randomizer Setup Guide
+
+## Required Software
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). Make sure to check the box for `SNI Client - Yoshi's Island Patch Setup`
```

Comment:
```suggestion
- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases).
```
You already know this one, just making it so you can put it in the batch commit.